File: 7KYU/Find the capitals/solution.cs
using System;
using System.Collections.Generic;
public static class Kata
{
  public static int[] Capitals(string word)
  {
      var capitals = new List<int>();
    
      for(var i = 0; i < word.Length; i++)
      {
         if(char.IsUpper(word[i]))
            capitals.Add(i);
      }
      return capitals.ToArray();       
  }
}

File: 7KYU/Find the capitals/solution.ts
export function capitals (word: string): number[] {
	var idxs = [];
  
  for(var i = 0; i < word.length; i++) {
    if(word[i] == word[i].toUpperCase()) {
      idxs.push(i);
    }
  }
  
  return idxs;
}